frontend/tic-tac-toe-app/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Is this not already covered by the root level .gitignore?

Reply:
it is. i just didnt see the duplicate! thank you!